proof/invariant-abstract/AARCH64/ArchVSpace_AI.thy

```diff
@@ -9,9 +9,6 @@
 AARCH64-specific VSpace invariants
 *)
 
-(* FIXME AARCH64 hyp: many VCPU/hyp-related lemmas from ARM_HYP pulled in because they exist in
-     ArchVSpace_AI there. However, it isn't clear where they should ultimately live. They might be
-     in the wrong order in this file (e.g. proved after they are needed). *)
```

Comment:
I vaguely remember us thinking about pulling the VCPU/hyp-related lemmas out of this file and into one of their own. Does removing this FIXME mean that we no longer want to do that?

Reply:
The main worry of the comment was that they are proved in the wrong order or mixed-in in a different way than ARM_HYP, which they weren't. It might still make sense to start a separate ArchVCPUAcc_AI file (the existing ArchVCPU_AI is only for the new vcpu invariant and is too high in the dependency tree).

We should probably open an issue about this, because if we do it, we should probably do it in both ARM_HYP and AARCH64 so that patches have a higher chance of applying to both.

Reply:
#571